README.md

```diff
@@ -283,6 +283,10 @@ var colorChangeTool: Tool {
 
 *CrashDetectionTool* sets up listeners on system exceptions which will be logged into a local json file with the stack trace. The tool is available on iOS.
 
+#### Database Import/Export tool
```

Comment:
Based on the proposed change, this name may change as well - if you'll agree on that. If not, it can stay like that.